aten/src/THC/generic/THCTensor.hpp

```diff
@@ -2,18 +2,7 @@
 #define THC_GENERIC_FILE "generic/THCTensor.hpp"
 #else
 
-typedef struct THCTensor
-{
-    int64_t *size;
-    int64_t *stride;
-    int nDimension;
-
-    THCStorage *storage;
-    ptrdiff_t storageOffset;
-    std::atomic<int> refcount;
-
-    char flag;
-
+typedef struct THCTensor : _THCTensor {
```

Comment:
Would be nice to have a comment here, but since it's dying tomorrow I guess it's fine :)